### ISSUE TITLE
CORS Header (Access-Control-Allow-Origin) is missing again

### ISSUE THREAD

**Issue by Steve-xmh** [DarrenOfficial/dpaste#242]
## Description

First of all, thank you for this free and available pastebins service!

I've noticed that #225 added header `Access-Control-Allow-Origin: *` that I can use for uploading some temporary data via javascript, but the header is missing now without reason:

![image](https://github.com/DarrenOfficial/dpaste/assets/39523898/e3d5ba65-99d1-436f-9645-42a495a40b71)

I know it will be a cash-burning work to deliver this free service. So if it's intentional I'll close this issue.

## Steps to reproduce

```javascript
const rawUrl = await fetch("https://dpaste.org/api/", {
    method: "POST",
    mode: "cors",
    headers: {
        "Content-Type": "application/x-www-form-urlencoded"
    },
    body: "format=url&expires=3600&filename=somedata.txt&content=testing"
});
```

**Comment by DarrenOfficial** [comment]
Hello, Yes, I've temporarily removed the CORS due to the way I implemented it. Due to how many request dpaste was getting, I was getting billed by Cloudflare per that extra request

![cors](https://github.com/DarrenOfficial/dpaste/assets/55406094/2b2051b6-c1cc-4237-b681-cbb6f8594ce8)

I must admit the way I add CORS @ Cloudflare edge was a bit stupid. I've pushed the changes I was planning on doing since that request was made. `Access-Control-Allow-Origin` is enabled again now, and the way I implemented it is way better, because I added it in the application layer :D

![New Config](https://github.com/DarrenOfficial/dpaste/assets/55406094/caeae903-7b6a-42e9-a7a9-e03f9a5ef1af)
